pkg/patch/sidecar.go

```diff
@@ -10,6 +10,7 @@ const envoyContainerTemplate = `
 {
   "name": "envoy",
   "image": "{{ .ContainerImage }}",
+  "imagePullPolicy": "Always",
```

Comment:
I think this will create unnecessary traffic and cost for App Mesh users, since EKS will pull from ECR on every pod create. I also think this would affect the startup time of all meshed pods considerably.

Reply:
Sure, I will take this out.